dashboard/test/logic/task_grid_filter_test.dart

```diff
@@ -165,6 +165,17 @@ void main() {
     });
   }
 
+  test('matches ios and android (replace) filters logic', () {
+    final TaskGridFilter iosFilter = TaskGridFilter.fromMap(<String, String>{'showiOS': 'true'});
+    final TaskGridFilter androidFilter = TaskGridFilter.fromMap(<String, String>{'showAndroid': 'true'});
+    final TaskGridFilter androidFalseFilter = TaskGridFilter.fromMap(<String, String>{'showAndroid': 'false'});
+
+    expect(iosFilter.matchesTask(QualifiedTask.fromTask(Task()..builderName = 'Mac_ios')), true);
```

Comment:
Test `Mac` and `Mac_ios` with showiOS true & showMacOS false, vice versa, both true, both false